src/components/layouts/Header.js

```diff
@@ -1,11 +1,35 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React, { useState } from 'react';
+import Notification from '../notification/Notification';
 
-export default class Header extends React.Component {
-  render() {
-    return (
+export default function Header() {
+  const [showNotification, setShowNotification] = useState(false);
+
+  const updateShowNotificationsScreen = () => {
+    setShowNotification(!showNotification);
+  };
+
+  return (
+    <>
```

Comment:
please make this `<div>` to avoid readability issues.

Reply:
Lets make use of fragments.